pkg/scheduler/plugins/drf/drf.go

```diff
@@ -95,6 +95,15 @@ func (drf *drfPlugin) OnSessionOpen(ssn *framework.Session) {
 		allocations := map[api.JobID]*api.Resource{}
 
 		for _, preemptee := range preemptees {
+
+			// Min part tasks in job should not be preempted
+			preempteeJob := ssn.Jobs[preemptee.Job]
+			readyNum := preempteeJob.ReadyTaskNum()
+			preemptable := preempteeJob.MinAvailable <= readyNum-1 || preempteeJob.MinAvailable == 1
```

Comment:
Abstract into a common function

Reply:
>preempteeJob := ssn.Jobs[preemptee.Job]
>readyNum := preempteeJob.ReadyTaskNum()

good suggestion. I thought to do it. The reson are
- only above two lines can be abstracted.
- only two places including drf/gang preemptFn use it. The reusablity is low.

Reply:
DRF should not care about `minAvailable`.

Reply:
Yes. pure DRF should only care about share. This is like adding another prerequisite(do not preempt not ready job) . However, this is the best way I can think of right now if we don't want to preempt min part tasks.